static/index.css

```diff
@@ -82,7 +82,7 @@ ul li {
     margin-top: 16px;
 }
 
-.content code {
+code {
```

Comment:
Lame attempt at fixing code blocks in blog, don't know if it worked as css cache is borked on my browser.

Reply:
Interesting.. would the code blocks not be under `.content`?

Reply:
Oh, this CSS doesn't even apply to the blogs.